test/k8sT/Services.go

```diff
@@ -1124,6 +1124,17 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 			}
 
 			if helpers.RunsOnGKE() {
+				k8s1ExternalIP, err := kubectl.GetNodeIPByLabel(helpers.K8s1, true)
+				Expect(err).Should(BeNil(), "Cannot retrieve Node External IP for"+helpers.K8s1)
```

Comment:
`"Cannot retrieve Node External IP for"+helpers.K8s1`

There's no space between `for` and `helpers.K8s1`, could you add spacing in between there?

Also, `Should*` accepts a format string as the second argument, and additional args. For ex:

```
Expect(err).Should(BeNil(), "Cannot retrieve Node External IP for %s", helpers.K8s1)
```

This should be preferred.

Reply:
Fixed.